package.json

```diff
@@ -33,9 +33,8 @@
   },
   "dependencies": {
     "chalk": "2.4.2",
-    "deasync": "0.1.15",
     "inquirer": "6.3.1",
-    "jhipster-core": "3.6.14",
+    "jhipster-core": "3.6.6",
```

Comment:
interesting, why?

Reply:
Because even if it’s a minor version api change. The values function. Has been removed. Other way. I copy it and use last dep

Reply:
I don't get it, what has changed exactly?

Reply:
https://github.com/jhipster/jhipster-core/commit/f25403d47333582d76396f2be774226424634b27#diff-237e749ebf2ad76c636e26520e5a81cdL48

Reply:
function named "values" has been removed

Reply:
Ah, just replace ObjectUtils.values by Object.values

Reply:
it's not ok what i have done?